src/ferui-components/tree-view/tree-node-component.ts

```diff
@@ -52,7 +54,7 @@ import { ScrollbarHelper } from '../utils/scrollbar-helper/scrollbar-helper.serv
         </span>
       </div>
     </div>
-    <div [style.margin-left.px]="calculatePadding() + 20">
+    <div [style.margin-left.px]="padding + 20">
```

Comment:
What is those 20px for again ? You shouldn't do any calculation on the template, this should be done in the controller instead. And you need to create a variable that explain why you've used this amount and add it directly to the "padding" variable.